c7n/tags.py

```diff
@@ -623,14 +626,26 @@ def validate(self):
             raise PolicyValidationError(
                 "Invalid timezone specified %s in %s" % (
                     self.tz, self.manager.data))
+
+        # Ensure that the msg writes a message in a format that marked-for-op will recognize.
+        # The expectation is that the string ends with ": op@date", so a simple regex to handle
+        # whitespace should be good.
+        msg = self.data.get('msg', self.default_template)
+        if not re.fullmatch(r".*: *{op}@{action_date} *", msg):
```

Comment:
while technically correct  and better validation, this also potentially causes some failures on extant policies for a new release due to tighter validation. we'll have to note that possibility in the release notes.

Reply:
I agree that we'll have to record it in the release notes. However if someone was using a `mark-for-op` with a custom message that didn't match this, they could never have it working with `marked-for-op`.